app/Resources/views/membership/register.html.twig

```diff
@@ -29,112 +29,132 @@
 
 {% block content %}
     <section class="l__wrapper--medium">
-        <div class="donation">
+        <div class="register">
             <h2>J'adhère au mouvement En Marche !</h2>
 
-            <div class="l__row l__row--top l__row--h-stretch">
-                <div class="donation__form">
-                    <form method="post" id="app-membership" class="form">
-                        {{ form_errors(form) }}
+            <p class="register--intro">
```

Comment:
If you really want to do integration, we use the BEM convention with the following synthax: `block__subblock--variant` :) .